mashup-example/using-build-tool/qlik-sense-desktop/configure.js

```diff
@@ -26,7 +26,9 @@ const nuked = embed.createConfiguration({
     language: 'en-US',
     keyboardNavigation: true,
     constraints: {
-      active: true, // do not allow interactions
+      active: false, // turn off interactions that affect the state of the visual representation like zoom, scroll, etc.
```

Comment:
change the comment

Reply:
how about "turn off interactions that affect the state of the visual representation including selection, zoom, scroll, etc"

Reply:
sounds good